pkg/controller/pxc/vs.go

```diff
@@ -1,120 +1,96 @@
 package pxc
 
 import (
-	"encoding/json"
 	"fmt"
 	"net/http"
 	"net/url"
-	"strings"
 	"time"
+
+	"github.com/percona/percona-xtradb-cluster-operator/versionserviceclient"
+	"github.com/percona/percona-xtradb-cluster-operator/versionserviceclient/models"
+	"github.com/percona/percona-xtradb-cluster-operator/versionserviceclient/version_service"
 )
 
-func (vs VersionServiceClient) GetExactVersion(vm versionMeta) (DepVersion, error) {
-	client := http.Client{
-		Timeout: 5 * time.Second,
-	}
+const productName = "pxc-operator"
 
-	requestURL, err := url.Parse(
-		fmt.Sprintf("%s/v1/pxc-operator/%s/%s",
-			strings.TrimRight(vs.URL, "/"),
-			vs.OpVersion,
-			vm.Apply,
-		),
-	)
+func (vs VersionServiceClient) GetExactVersion(vm versionMeta) (DepVersion, error) {
+	requestURL, err := url.Parse(vs.URL)
 	if err != nil {
 		return DepVersion{}, err
```

Comment:
please, wrap error message with more context. Here and in all other places

Reply:
8578c8d76d9a1711eabd53fe7fcf4be3f840e62b